scripts/package.sh

```diff
@@ -14,7 +14,11 @@ else
 fi
 
 npm run clean
-npm run build$dev
+# riot-web is at `workspace`.
+# matrix-react-sdk is at `workspace/node_modules/matrix-react-sdk`, but this is
+# symlinked to a repo at `workspace/matrix-react-sdk`.
+# To get from `workspace/matrix-react-sdk/lib` up to the lang file, we use:
```

Comment:
not sure if symlinks are considered when resolving the parent directory?

Reply:
Right, I am also not sure myself... Looking at the [Jenkins log](https://matrix.org/jenkins/job/VectorWebDevelop/4848/console), it says things like:

```
Error: Can't resolve '../../riot-web/webapp/i18n/languages.json' in '/var/lib/jenkins/jobs/VectorWebDevelop/workspace/matrix-react-sdk/lib'
```

which suggests to me that it _is_ resolving the symlink, and so we need to navigate from the symlink target. I'm not sure how to know for sure other than just trying... If it still fails, I'll try it the other way, I guess. 😖

Reply:
wfm